frontend/pages/reports/index.js

```diff
@@ -0,0 +1,227 @@
+import s from 'styled-components'
+import Router from 'next/router'
+import { useEffect, useState } from 'react'
+import renderPage from '../../renderPage.js'
+import { Icon, Empty, Checkbox, CheckboxLabel } from '../../components/common'
+import { doApiRequest, API_BASE_URL } from '../../utils'
+import { Container } from '../../components/common/Container'
+import { CLUBS_GREY } from '../../constants/colors'
+import Edit from './edit'
+
+const GroupLabel = s.h4`
+  margin-bottom: 0em !important;
+  font-size: 32px;
+  color: #626572;
+`
+
+const ColoredHeader = s.div`
+  background: linear-gradient(to right, #ef4c5f, #4954f4);
```

Comment:
same, use constant colors if they exist

Reply:
^can also add the constant colors so other ppl can use them and they're more human readable